src/avatar.scss

```diff
@@ -40,7 +40,7 @@ $block: #{$fd-namespace}-avatar;
     [class*="#{$block}"] {
       @include fd-reset();
 
-      color: var(--sapContent_ContrastTextColor);
+      color: var(--fdAvatar_DefaultColor);
```

Comment:
I think this should stay ` --sapContent_ContrastTextColor`. It's specified for all themes for the default color - this should be white in hcb and black in hcw. We've got the opposite happening.